src/main/java/com/itschool/eventmanagment/controllers/EventControler.java

```diff
@@ -1,21 +1,33 @@
 package com.itschool.eventmanagment.controllers;
 
 import com.itschool.eventmanagment.models.dtos.EventDTO;
+import com.itschool.eventmanagment.models.entities.Event;
 import com.itschool.eventmanagment.services.EventService;
 import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.bind.annotation.RestController;
 
+import java.time.LocalDateTime;
+import java.util.List;
+
 @RestController
 public class EventControler {
     private final EventService eventService;
 
     public EventControler(EventService eventService) {
         this.eventService = eventService;
     }
+
     @PostMapping("/api/events")
-    public ResponseEntity<EventDTO> createEvent(@RequestBody EventDTO eventDTO){
+    public ResponseEntity<EventDTO> createEvent(@RequestBody EventDTO eventDTO) {
         return ResponseEntity.ok(eventService.createEvent(eventDTO));
     }
+
+    @GetMapping("/api/events")
+    public ResponseEntity<List<EventDTO>> searchEvent(@RequestParam(required = false) LocalDateTime from, @RequestParam(required = false) LocalDateTime to, @RequestParam(required = false) String location, @RequestParam(required = false) String sortBy) {
```

Comment:
We can leverage `@DateTimeFormat` annotation here:

```suggestion
    public ResponseEntity<List<EventDTO>> searchEvent(@RequestParam(required = false) @DateTimeFormat(iso = DateTimeFormat.ISO.DATE_TIME) LocalDateTime from, @RequestParam(required = false) LocalDateTime to, @RequestParam(required = false) String location, @RequestParam(required = false) String sortBy) {
```

**Code formatting:**
- when there are multiple parameters, please consider to format the code so we can have one parameter under the other one, to improve visibility


```
public ResponseEntity<List<EventDTO>> searchEvent(@RequestParam(required = false) LocalDateTime from, 
                                                  @RequestParam(required = false) LocalDateTime to, 
                                                  @RequestParam(required = false) String location, 
                                                  @RequestParam(required = false) String sortBy) {
```

**Method naming:**
- since we return a list of DTOs, we can/should use plural

**DateTime format:**
- in terms of response formatting, it's generally a good practice to follow the ISO 8601 standard, which looks like: `2024-10-10T12:00:00`
- we can ensure the correct format in the DTO by annotation the LocalDateTime field with:

```
@JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd'T'HH:mm:ss")
private LocalDateTime dateTime;
```
I've noticed the response you had and also the code formatting in the code for date.

<img width="667" alt="image" src="https://github.com/user-attachments/assets/e1cc3843-a73a-4111-a755-dacb294d41b2">